cookiecutter.json

```diff
@@ -1,15 +1,16 @@
 {
-    "full_name": "Antoine Legrand",
-    "email": "2t.antoine@gmail.com",
-    "github_username": "ant31",
-    "project_name": "newproj",
+    "full_name": "",
+    "email": "",
+    "github_username": "",
+    "project_name": "",
     "project_slug": "{{cookiecutter.project_name}}",
-    "varEnvPrefix": "{{cookiecutter.project_slug | upper}}",
-    "baseclass": "{{ cookiecutter.project_slug | capitalize}}",
+    "package_name": "{{cookiecutter.project_slug.lower().replace(' ', '_').replace('-', '_')}}",
```

Comment:
what's the difference between package_name and slug ?

In which case should we use slug, and in which case package_name?

Reply:
in short, what you call package_name is what it's called project_slug, following cookiecutter author list of variable : "https://github.com/audreyfeldroy/cookiecutter-pypackage/blob/master/cookiecutter.json

```
  "project_name": "Python Boilerplate",
  "project_slug": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
```

If there is no two usecases, we should keep only one. I'm fine with package_name and removing project_slug